### ISSUE TITLE
Midterm Report Peer Review by cct65

### ISSUE THREAD

**Issue by carotroude** [vanshikabansal/ORIE-4741#8]
The objective of the project is to determine which horse will win under a specific set of racing conditions, and to potentially create a long-term betting strategy. I think it is great as it has a real economic value. The final product of your second approach could be a strategic tool to set rewarding betting strategies.

I really like the fact that there is insightful thought given to how you are selecting and transforming your variables according to the specific objective that you are trying to reach. I also really appreciate your data visualizations; it gives a more concrete view of how your data are distributed and correlated. It is a good thing that you decided to implement random forest, in this way you are correcting the overfitting habit of simple decision trees.

I have a few comments; first of all, I do not really understand how you selected your features. Intuition can help in many situations but here I would advised you to include more features in the beginning because you do not really have an evidence of their impact on your model. Some features may be very interesting and worth being included in your final dataset. In order to make a good choice you could try forwards or backwards variable subset selection. 
I am a little bit confused about your train and test set choice. I think that you should allow more data for you train set than for your test set at this stage of the project or you can also use cross-validation to test your model. 

Overall I think that it is a really interesting project, I look forward to seeing your progress!
